src/CONST.ts

```diff
@@ -2899,6 +2899,7 @@ const CONST = {
     },
     TRANSLATION_KEYS: {
         ATTACHMENT: 'common.attachment',
+        HIDDEN_MESSSAGE: 'parentReportAction.hiddenMessage',
```

Comment:
```suggestion
```

Reply:
I think we missed removing this - we don't use this anywhere right?